### DISCUSSION TITLE
Preview multiple entries at a time

### DISCUSSION THREAD

**Discussion by weotch** [craftcms/cms#10818] [Ideas] [answered]
It would be great if we could preview multiple entries at a time.  For example, if you have a product page with "related products" and you want to preview changes to the main product page as well as to some of the related product cards.

UX-wise, I'm thinking that from the Live Preview view's header you would add a filterable select menu that allows you to add additional drafts to the active preview.

**Comment by brandonkelly** [comment, marked as answer]
This was already suggested here: #8941
